src/um/winnt.rs

```diff
@@ -2123,27 +2043,28 @@ STRUCT!{struct ACL_SIZE_INFORMATION {
     AclBytesFree: DWORD,
 }}
 pub type PACL_SIZE_INFORMATION = *mut ACL_SIZE_INFORMATION;
-pub const SECURITY_DESCRIPTOR_REVISION: DWORD = 1;
-pub const SECURITY_DESCRIPTOR_REVISION1: DWORD = 1;
-#[cfg(target_arch = "x86_64")]
-pub const SECURITY_DESCRIPTOR_MIN_LENGTH: usize = 40;
-#[cfg(target_arch = "x86")]
-pub const SECURITY_DESCRIPTOR_MIN_LENGTH: usize = 20;
+pub const SECURITY_DESCRIPTOR_REVISION: BYTE = 1;
+pub const SECURITY_DESCRIPTOR_REVISION1: BYTE = 1;
+#[cfg(target_arch="x86")]
+const __SIZE_OF_SECURITY_DESCRIPTOR: SIZE_T = 20;
+#[cfg(target_arch="x86_64")]
+const __SIZE_OF_SECURITY_DESCRIPTOR: SIZE_T = 40;
+pub const SECURITY_DESCRIPTOR_MIN_LENGTH: SIZE_T = __SIZE_OF_SECURITY_DESCRIPTOR;
```

Comment:
Don't add the fake `__SIZE_OF` constants. Just hardcode the size into `SECURITY_DESCRIPTOR_MIN_LENGTH` with a `// FIXME` to fix it later when `sizeof` is a const function.